bot/src/modules/birthday/commands/birthday/difference.ts

```diff
@@ -49,13 +48,18 @@ export default SubCommand(
 
     if (!second_user) second_user = msg.member as GuildMember;
 
-    const users = await state.db.birthday.findMany({
+    const users = await state.db.user.findMany({
       where: {
-        OR: [{ userId: first_user.id }, { userId: second_user.id }],
-        guildId: msg.guildId,
-        NOT: { date: null },
+        OR: [{ id: first_user.id }, { id: second_user.id }],
+        birthdate: { not: null },
+        UserGuildConfig: {
+          some: {
+            guildId: msg.guild.id,
+            birthdayEnabled: true,
```

Comment:
Would having some way to have separate "i want to get the announcement and role" and "i want for others to be able to look it up but not have it announced" modes be a good idea?